index.js

```diff
@@ -0,0 +1,223 @@
+    id: 0,
+    header: 'Tonic',
+    canopy: 'CANOPY',
+    backend: 'Back End Dev',
+    year: '2015',
+    popupPhoto: 'media/Snapshoo-desktop.svg',
+    content: 'Lorem Ipsum is dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text since the 1500s when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries.',
+    tools: ['HTML', 'CSS', 'JavaScript'],
+  },
+
+  {
+    id: 1,
+    header: 'Multi-Post Stories',
+    canopy: 'CANOPY',
+    backend: 'Back End Dev',
+    year: '2015',
+    popupPhoto: 'media/Snapshoot Portfolio.svg',
+    content: 'Lorem Ipsum is dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text since the 1500s when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries.',
+    tools: ['HTML', 'CSS', 'JavaScript'],
+  },
+
+  {
+    id: 2,
+    header: 'Facebook 360',
+    canopy: 'FACEBOOK',
+    backend: 'Full Stack Dev',
+    year: '2015',
+    popupPhoto: 'media/Snapshoot Portfolio2.svg',
+    content: 'Lorem Ipsum is dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text since the 1500s when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries.',
+    tools: ['HTML', 'Ruby on rails', 'CSS', 'JavaScript'],
+  },
+
+  {
+    id: 3,
+    header: 'Multi-Post Stories',
+    canopy: 'CANOPY',
+    backend: 'Back End Dev',
+    year: '2015',
+    popupPhoto: 'media/Snapshoot Portfolio3.svg',
+    content: 'Lorem Ipsum is dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text since the 1500s when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries.',
+    tools: ['HTML', 'CSS', 'JavaScript'],
+  },
+];
+
+projects.innerHTML = `
+<div class="card-body">
+    <div class="card-img">
+        <img src="${projectInfo[0].popupPhoto}" width="100%" />
+    </div>
+    <div>
+        <h2 class="card-header">${projectInfo[0].header}</h2>
+        <ul class="card-sub">
+      <li class="remove">${projectInfo[0].canopy}</li>
+      <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>
+      <li>${projectInfo[0].backend}</li>
+      <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>      
+      <li>${projectInfo[0].year}</li>
+    </ul>
+    <div class="card-content">
+      <p>
+      ${projectInfo[0].content}
+      </p>
+      <ul class="tools">
+        <li>${projectInfo[0].tools[0]}</li>
+        <li>${projectInfo[0].tools[1]}</li>
+        <li>${projectInfo[0].tools[2]}</li>
+      </ul>
+      <button class="popBtn show">See project</button>
+    </div>
+  </div>
+  </div>
```

Comment:
- [ ] Kindly note that repeating sections for each popup window is what we are trying to prevent in the `index.html` by putting the items into the array, which is what has ended up happening in the `index.js` file, imagine having one hundred array of objects for popup window, something like your GitHub field, kindly follow the DRY principle by looping through the array and return a single section which will loop through the array in producing the rest, you can loop through the array with a for loop, foreach or map, research and choose which best suits you.